synapse/lib/cell.py

```diff
@@ -2151,3 +2158,30 @@ async def kill(self, user, iden):
         perm = '.'.join(perm)
         raise s_exc.AuthDeny(mesg=f'User must have permission {perm} or own the task',
                              task=iden, user=str(user), perm=perm)
+
+    async def getCellVersionInfo(self):
```

Comment:
If we did go with getCellInfo() we could also stuff a bunch of the future utilization/system info in here as well. :)